osquery/tables/system/windows/logon_sessions.cpp

```diff
@@ -0,0 +1,142 @@
+// clang-format off
+#include <Windows.h>
+#include <NTSecAPI.h>
+#include <sddl.h>
+#include <tchar.h>
+#include <vector>
+
+#include <osquery/core.h>
+#include <osquery/logger.h>
+#include <osquery/tables.h>
+
+#include "osquery/core/windows/process_ops.h"
+// clang-format on
+
+#define CHECK_BIT(var, pos) ((var) & (1 << (pos)))
+#define JAN_1_1601_TO_JAN_1_1970 116444736000000000
+#define HUNDREDNANOSECONDS_TO_SECONDS 10000000
+
+namespace osquery {
+namespace tables {
+LONGLONG filetimeToUnixtime(const LARGE_INTEGER& ft) {
+  LARGE_INTEGER date, adjust;
+  date.HighPart = ft.HighPart;
+  date.LowPart = ft.LowPart;
+  adjust.QuadPart = JAN_1_1601_TO_JAN_1_1970;
+  date.QuadPart -= adjust.QuadPart;
+  return date.QuadPart / HUNDREDNANOSECONDS_TO_SECONDS;
+}
+
+LPWSTR GetLogonType(ULONG logonType) {
+  switch (logonType) {
+  case 2:
+    return L"Interactive";
+  case 3:
+    return L"Network";
+  case 4:
+    return L"Batch";
+  case 5:
+    return L"Service";
+  case 6:
+    return L"Proxy";
+  case 7:
+    return L"Unlock";
+  case 8:
+    return L"NetworkCleartext";
+  case 9:
+    return L"NewCredentials";
+  case 10:
+    return L"RemoteInteractive";
+  case 11:
+    return L"CachedInteractive";
+  case 12:
+    return L"CachedRemoteInteractive";
+  case 13:
+    return L"CachedUnlock";
+  default:
+    return L"None";
+  }
+}
+
+QueryData QueryLogonSessions(QueryContext& context) {
+  QueryData results;
+  bool loadedManually = false;
```

Comment:
nit: we've started drawing a line in the sand, and are requiring local variables follow posix in being `snake_case`. Would you mind updating your code base to reflect? Sorry for the inconsistency in our code base :)